src/react-apps/applications/shared/src/components/AltinnLogo.tsx

```diff
@@ -0,0 +1,44 @@
+                        <path d='M19.8555,19.7446667 C16.905,19.7446667 15.6636667,17.6761667 15.6636667,15.6356667 L15.6636667,0.110833333 L19.2488333,0.110833333 L19.2488333,15.4151667 C19.2488333,16.2703333 19.6081667,16.7113333 20.5181667,16.7113333 L21.8971667,16.7113333 L21.8971667,19.7446667 L19.8555,19.7446667 L19.8555,19.7446667 Z' id='Shape' />
+                        <path d='M28.5693333,19.7446667 C25.6468333,19.7446667 24.4055,17.6761667 24.4055,15.6356667 L24.4055,8.52133333 L22.8888333,8.52133333 L22.8888333,5.79133333 L24.4055,5.79133333 L24.4055,1.54466667 L27.9906667,1.54466667 L27.9906667,5.79133333 L30.527,5.79133333 L30.527,8.52133333 L27.9906667,8.52133333 L27.9906667,15.4151667 C27.9906667,16.2423333 28.3768333,16.7113333 29.232,16.7113333 L30.5281667,16.7113333 L30.5281667,19.7446667 L28.5693333,19.7446667 L28.5693333,19.7446667 Z' id='Shape' />
+                        <path d='M33.0353333,2.92366667 L33.0353333,0 L36.7033333,0 L36.7033333,2.92366667 L33.0353333,2.92366667 L33.0353333,2.92366667 Z M33.0913333,19.7446667 L33.0913333,5.37716667 L36.6765,5.37716667 L36.6765,19.7446667 L33.0913333,19.7446667 L33.0913333,19.7446667 Z' id='Shape' />
+                        <path d='M48.3396667,19.7446667 L48.3396667,11.0576667 C48.3396667,9.1 47.0983333,8.43733333 45.969,8.43733333 C44.8385,8.43733333 43.5691667,9.09883333 43.5691667,11.0576667 L43.5691667,19.7435 L39.984,19.7435 L39.984,5.37716667 L43.4863333,5.37716667 L43.4863333,6.70016667 C44.4231667,5.70733333 45.7473333,5.2115 47.0715,5.2115 C48.5053333,5.2115 49.6638333,5.6805 50.463,6.47966667 C51.6203333,7.637 51.9248333,8.98916667 51.9248333,10.5606667 L51.9248333,19.7435 L48.3396667,19.7435 L48.3396667,19.7446667 Z' id='Shape' />
+                        <path d='M63.4491667,19.7446667 L63.4491667,11.0576667 C63.4491667,9.1 62.2078333,8.43733333 61.0785,8.43733333 C59.948,8.43733333 58.6786667,9.09883333 58.6786667,11.0576667 L58.6786667,19.7435 L55.0935,19.7435 L55.0935,5.37716667 L58.5958333,5.37716667 L58.5958333,6.70016667 C59.5326667,5.70733333 60.8568333,5.2115 62.181,5.2115 C63.6148333,5.2115 64.7733333,5.6805 65.5725,6.47966667 C66.7298333,7.637 67.0343333,8.98916667 67.0343333,10.5606667 L67.0343333,19.7435 L63.4491667,19.7435 L63.4491667,19.7446667 Z' id='Shape' />
+                    </g>
+                    <g id='logoelement'>
+                        <rect id='Rectangle-35' x='40' y='0' width='12' height='4' />
+                        <rect id='Rectangle-35' x='40' y='8' width='12' height='4' />
+                        <rect id='Rectangle-35' x='40' y='16' width='12' height='4' />
```

Comment:
Should svg-icons be created as react components? I feel like this is more of a static resource? Or are there any decisions I'm not aware of?

Reply:
Needed to change the fill color of the logo depending on "state", so made it a react component.
It's not a discussed decision and I'm open to other solutions :)

Reply:
Unsure how important this is to fix, but I would prefer something like 

<svg>
    <image class="change-my-color" xlink:href="https://somelink.com/file.svg" src="ppngfallback.png" />
</svg>

Then you can style the image either using the css class or inline styling. But Ill approve the PR, maybe @matsgm or @jezpoz have any thoughts?

Reply:
#2103 : issue on removing SVG-icons to CDN